### DISCUSSION TITLE
time filter

### DISCUSSION THREAD

**Discussion by kxleee** [milvus-io/milvus#38670] [Q&A and General discussion]
I have a standard time field in my data field, how do I filter the data by time, for example, I want to filter the data between the start time and the end time, and sort it

**Comment by yhmo** [comment]
Currently, milvus doesn't support "Datetime" type field. You can turn the time value to timestamp(long type), and compare the time by long value.

**Reply by kxleee** [reply]
> Currently, milvus doesn't support "Datetime" type field. You can turn the time value to timestamp(long type), and compare the time by long value.

Thanks your reply.  Is there any plan to support "Datetime" in the future? We are considering moving from elasticsearch to milvus.

**Reply by yhmo** [reply]
"Datetime" type field is a valuable feature. Wait @xiaofan-luan to comment.

**Reply by xiaofan-luan** [reply]
Agree that datetime is valuable.
Date time is in our roadmap but not have enough engineering power.
Any volunteers is welcomed
